src/components/game/table/Game.js

```diff
@@ -1,44 +1,56 @@
-import React, { useContext, useEffect } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { GlobalContext } from '../../../context/GlobalState';
-import { Container, useApp } from '@inlet/react-pixi';
+import { Container, useTick } from '@inlet/react-pixi';
 import { Board } from './Board';
-import { useAspectRatioContainer } from '../../../hooks/aspectRatioContainer';
-import { useTableSettuper } from '../../../hooks/tableSettuper';
-import { useImages } from '../../../hooks/images';
+import { useTableSettuper } from './useTableSettuper';
+import { useStacks } from '../stacks/useStacks';
+import { useImages } from './useImages';
 import { PlayerPanel } from './PlayerPanel';
 import { GameObject } from '../game-object/GameObject';
+import { Stack } from '../stacks/Stack';
+import { useScreenResizer } from './useScreenResizer';
 
 export const Game = ({ players }) => {
-  const app = useApp();
-  const { screenDefaults, gameObjects } = useContext(GlobalContext);
-  const { width, height } = useAspectRatioContainer(screenDefaults.aspectRatio);
+  const { gameObjects, stacks } = useContext(GlobalContext);
+  const [gameObjectIds, setGameObjectIds] = useState(Object.keys(gameObjects));
 
   const images = useImages();
+  useScreenResizer();
   const { spawnDiamondWithTile, spawnPlayerWithTile } = useTableSettuper();
+  const { createStacks, getContentForAllStacks } = useStacks();
 
   useEffect(() => {
+    // only if(host) and then send gameObjects and stacks to rest players
```

Comment:
Hosta już nie ma, chodzi Ci o tego co stworzył grę?

Reply:
tak, to jest host wlasnie, nie ma hosta serwera, ale host jest